src/Controls/Maps/src/Pin.cs

```diff
@@ -40,22 +41,20 @@ public PinType Type
 		}
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public object MarkerId
+		public object? MarkerId
```

Comment:
This is part of a public interface, it doesn't need EditorBrowsable.Never.

Reply:
Fixed